.github/workflows/tests.yml

```diff
@@ -200,6 +225,10 @@ jobs:
   multi_arch_support:
     name: Build iOS App for Multiple Architecture
     runs-on: macos-12
+    env:
+      # i386 was removed on rules_apple 3.x.x - this test case needs reworking
+      # to exemplify fat binaries on the latest version
```

Comment:
Nit: Why is this comment in the Bazel version line? Potentially confusing as it looks like an issue with the version of Bazel itself? Maybe move it up?

Reply:
Ok fair enough will move it up then